samples/example-vue-site/core/src/main/java/com/example/site/models/NavModel.java

```diff
@@ -63,8 +63,8 @@ public String getBrand() {
         List<NavItem> ret = new ArrayList<>();
 
         RenderContext rx = PageMerge.getRenderContext();
-        SlingHttpServletRequest request = rx.getRequest();
-        Resource homePage = getResourceAt(request.getResource(), 3);
+//        SlingHttpServletRequest request = rx.getRequest();
```

Comment:
rather remove it than pollute the code with comments

Reply:
I removed the comment and unused code